spec/03-plugins/14-request-transformer/02-access_spec.lua

```diff
@@ -166,7 +197,7 @@ for _, strategy in helpers.each_strategy() do
     end)
 
     lazy_teardown(function()
-      helpers.stop_kong()
+      helpers.stop_kong(nil, true, true)
```

Comment:
This helper only accepts 2 arguments as of the current master branch. Also, better not to modify the default behavior for debug purposes in the PR itself